### PR TITLE
chore: increment coverage

### PR DESCRIPTION
**Pull request by shuffledex**

### CODE REVIEW

**Review by monitz87 — CHANGES_REQUESTED** [review]

File: src/utils/__tests__/conversion.ts [hunk 1]
```diff
@@ -1291,6 +1417,53 @@ describe('permissionsToMeshPermissions and meshPermissionsToPermissions', () =>
...
+
+    fakeResult = {
+      tokens: {
+        values: [entityMockUtils.getSecurityTokenInstance({ ticker })],
+        type: PermissionType.Exclude,
+      },
+      transactions: {
+        type: PermissionType.Exclude,
+        values: [],
+        exceptions: [TxTags.identity.AddClaim],
```
[the remark this answers is not in the document]
Comment (shuffledex) [reply src/utils/__tests__/conversion.ts:1429]: sounds I made this case to cover a line. If I add
`values: [ModuleName.Identity],`
the result is:
`"transactions":{"type":"Exclude","values":[],"exceptions":["identity.addClaim"]}`
and the test fails

File: src/utils/conversion.ts [hunk 2]
```diff
@@ -753,20 +753,21 @@ export function permissionsToMeshPermissions(
 
           pallet.tx.push(dispatchableName);
         } else {
-          const pallet = extrinsicDict[tag];
+          const upperTag = stringUpperFirst(tag);
```
[the remark this answers is not in the document]
Comment (shuffledex) [reply src/utils/conversion.ts:756]: `extrinsicDict[palletName]`
palletName comes from splitTag that converts dispatchableName in upper case.
If a ModuleName is the first element of the array, the key must be in upper case as well to maintain compatibility

[the remark this answers is not in the document]
Comment (monitz87) [reply src/utils/conversion.ts]: TxTag dispatchable names are in lowercase

**Review by monitz87 — CHANGES_REQUESTED** [review]

File: src/utils/conversion.ts [hunk 3]
```diff
@@ -929,7 +930,7 @@ export function meshPermissionsToPermissions(
     transactions = {
       // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
       type: extrinsicType!,
-      values: txValues,
+      values: uniq(txValues),
```
[the remark this answers is not in the document]
Comment (shuffledex) [reply src/utils/conversion.ts:933]: if Except has more than one tx from the same module name, I avoid the duplicates

[the remark this answers is not in the document]
Comment (monitz87) [reply src/utils/conversion.ts:933]: You're only adding the module name once per pallet to `txValues`

[the remark this answers is not in the document]
Comment (shuffledex) [reply src/utils/conversion.ts]: done

**Review by monitz87 — APPROVED** [review]

**Pull request merged**